src/rpc/util.h

```diff
@@ -243,6 +243,7 @@ struct RPCResult {
         ARR_FIXED,  //!< Special array that has a fixed number of entries
         NUM_TIME,   //!< Special numeric to denote unix epoch time
         ELISION,    //!< Special type to denote elision (...)
+        EMPTY,      //!< Special type to allow empty OBJ
```

Comment:
pico-nit, wonder if `EMPTY_OBJ` would be clearer (feel free to ignore)
```suggestion
        EMPTY_OBJ,      //!< Special type to allow empty JSON object
```

Reply:
I called it `OBJ` to emphasise its relation to that enum.

Reply:
Maybe, @MarcoFalke?

Reply:
OBJ_EMPTY :ballot_box:

Reply:
`OBJ_EMPTY` it is!